data/campaigns/Eastern_Invasion/scenarios/12_Evacuation.cfg

```diff
@@ -172,185 +268,625 @@
+                            [endlevel]
+                                result=defeat
+                            [/endlevel]
+                            [return][/return]
+                        [/then]
+                    [/if]
+                    {CLEAR_VARIABLE found_unit}
+
+                    #-----------------
+                    # UNITS LEFT BEHIND
+                    #-----------------
+                    {PUT_TO_RECALL_LIST (
+                        side=1
+                        {NOT_ACROSS_RIVER}
+                    )}
+
+                    # if we have a meaningful amount of leveled units, play a brief cutscene
+                    {STORE_RECALLS (level=2-99) left_behind_units} # first just store the 2-99 level units, so we can check the count
+                    {STORE_RECALLS (level=2-99) gweddrys_remnants} # save these units in a new variable, which we'll use for S20 (the original variable gets modified)
+                    [if] {VARIABLE_CONDITIONAL left_behind_units.length greater_than_equal_to 2}
+                        [then]
+                            {STORE_RECALLS (level=0-1) left_behind_units} # add the 0-1 units in too
+                            {STORE_RECALLS (level=0-1) gweddrys_remnants}
+                            [store_locations]
+                                x=24,25,24,23,  26,25,23,  23,21,23 # assorted tiles near your starting keep
+                                y=10,11, 9,10,  10,13, 9,  11,10,12
+                                variable=recall_locations
+                            [/store_locations]
+                            [foreach]
+                                array=recall_locations
+                                [do]
+                                    [recall]
+                                        find_in=left_behind_units[0]
+                                        x,y=$this_item.x,$this_item.y
+                                    [/recall]
+                                    {CLEAR_VARIABLE left_behind_units[0]}
+                                [/do]
+                            [/foreach]
+
+                            [message]
+                                speaker=Owaec
+                                message= _ "Alas, what have I done? So many brave soldiers of Wesnoth, left behind in the wildlands..."
+                            [/message]
+                            [message]
+                                side=1
+                                x,y=21-26,9-13 # remember that 'radius' isn't supported in a SUF
+                                level=2-99
+
+                                [not]
+                                    id=Hahid al-Ali
+                                [/not]
+                                # would be very out of character for him
+                                message= _ "At least you three commanders are safe. From here, we must ride swift, ride strong, and ride to save all of Wesnoth!"
```

Comment:
'ride swiftly'

Reply:
That change would render it inconsistent with the following "ride strong". I think it's fine as is.

Reply:
If it was poetry. The consistency doesn't justify it for me, but whatever.

Reply:
I agree with celmin; it's ok to have moments of poeticness even in prose

Reply:
Especially when it's a motivational speech or the like.